.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
can we publish a surikat eslint project in which all projects can depend on for the rules. Something like.

"You can also extend configurations using shareable configuration packages. To do so, be sure to install the configuration package you want from npm and then use the package name, such as:

{
    "extends": "eslint-config-myrules",

    "rules": {
        // Override any settings from the "parent" configuration
        "eqeqeq": 1
    }
}
In this example, the eslint-config-myrules package will be loaded as an object and used as the parent of this configuration.

Note: You can omit eslint-config- and ESLint will automatically insert it for you, similar to how plugins work. See Shareable Configs for more information."